README.md

```diff
@@ -42,6 +46,10 @@ Anyone with the appropriate credentials can "cut a release" of jackdaw using the
 
 Steps 2 to 6 is essentially `git tag $version -m "$title\n\n$description" && git push --tags`
 
+#### Snapshot release
+
+Snapshot releases can be created by pushing a tag with the format `publish-snapshot-semver`
```

Comment:
Does this mean anyone can publish a snapshot release with any version they want?

Reply:
Described slightly higher in the README, essentially:
* outside contributors cannot tag, can only open PRs from forks
* tags are protected
* deploy env is protected, deploy step cannot run without approval from jackdaw or credit-applications team review